src/scripts/main.js

```diff
@@ -1,3 +1,58 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const tree = document.querySelector('.tree');
+
+  function labelTree(item) {
+    console.log(item);
+
+    if (!item) {
+      return;
+    }
+
+    if (!item.children.length) {
+      return;
+    };
```

Comment:
You can simplify it to not repeat the code 
```suggestion
    if (!item?.children?.length) {
      return;
    }
```
Using the optional chaining operator (?.) provides a concise way to handle cases where certain properties or nested objects may not exist. It avoids errors by gracefully handling situations where the properties are not accessible.